app/script/calling/entities/CallEntity.js

```diff
@@ -135,7 +134,9 @@ z.calling.entities.CallEntity = class CallEntity {
       return false;
     });
 
-    this.participantsCount = ko.pureComputed(() => this.getNumberOfParticipants(this.selfUserJoined()));
+    ko.pureComputed(() => this.getNumberOfParticipants(this.selfUserJoined())).subscribe(usersInCall => {
+      this.telemetry.numberOfParticipantsChanged(usersInCall);
+    });
```

Comment:
This is the only place where we use `getNumberOfParticipants`. Let turn it into an observable and use it.